src/collections/_documentation/workflow/user-settings.md

```diff
@@ -0,0 +1,62 @@
+---
+title: User Settings
+sidebar_order: 8
+---
+Manage your personal account details from the User Settings page, available in the dropdown menu under your organization’s name.
+
+## Account Details
+
+All changes in this section are saved automatically.
+
+## Name
+
+Change the name displayed next to your avatar by editing the Name field.
+
+[{% asset user-settings/us-name.png alt="Name section in User Settings" %}]({% asset user-settings/us-name.png @path %})
+
+## Preferences
+
+### Stack trace Order
+
+Choose the default ordering of frames in stack traces. By default, this order is set to let Sentry decide. 
+
+[{% asset user-settings/us-sto.png alt="Stack trace order section in User Settings" %}]({% asset user-settings/us-sto.png @path %})
+
+### Language
+
+Choose the default language used in your Dashboard from the dropdown.
```

Comment:
Should we use another word instead of "Dashboard"? Mostly because we have the "Dashboards" feature.

Reply:
Changed to: "Choose the default language used by Sentry from the dropdown."